src/circuit/tx.rs

```diff
@@ -144,14 +148,22 @@ pub fn c_transfer<C:CS, P:PoolParams<Fr=C::Fr>>(
     let out_note_hash = s.tx.output.1.iter().map(|e| e.hash(params)).collect::<Vec<_>>();
     let out_hash = [[out_account_hash].as_ref(), out_note_hash.as_slice()].concat();
 
-    //assert out notes are unique or zero
+    //assert out notes are unique or zero, check min value, compute out sum
+    let mut out_notes_sum: CNum<C> = p.derive_const(&Num::ZERO);
     let mut t:CNum<C> = p.derive_const(&Num::ZERO);
     let mut out_note_zero_num:CNum<C> = p.derive_const(&Num::ZERO);
     for i in 0..OUT {
-        out_note_zero_num+=s.tx.output.1[i].is_zero().as_num();
+        let out_note_is_zero = s.tx.output.1[i].is_zero();
+        let is_equal_or_greater_than_min = !c_comp(&p.out_note_min.as_num(), &s.tx.output.1[i].b.as_num(), BALANCE_SIZE_BITS);

```

Comment:
Imagine a user with a single input note of 10 BOB, while the min limit for output note is configured at 1 BOB.
How can he make a transfer of 9.5 BOB?